ngraph/python/src/pyngraph/frontend_manager.cpp

```diff
@@ -101,3 +103,38 @@ void regclass_pyngraph_FEC(py::module m)
            const ngraph::frontend::FrontEndCapabilities& b) { return a == b; },
         py::is_operator());
 }
+
+void regclass_pyngraph_ErrorCode(py::module m)
+{
+    py::class_<ngraph::frontend::ErrorCode, std::shared_ptr<ngraph::frontend::ErrorCode>> type(
+        m, "ErrorCode");
+    type.attr("ERROR_GENERAL") = ngraph::frontend::ErrorCode::ERROR_GENERAL;
+    type.attr("NOT_IMPLEMENTED") = ngraph::frontend::ErrorCode::NOT_IMPLEMENTED;
+    type.attr("OP_VALIDATION_FAILED") = ngraph::frontend::ErrorCode::OP_VALIDATION_FAILED;
+    type.attr("INITIALIZATION_ERROR") = ngraph::frontend::ErrorCode::INITIALIZATION_ERROR;
```

Comment:
small minor inconsistency: "ERROR_GENERAL" and "INITIALIZATION_ERROR". Could be either "GENERAL_ERROR" or "ERROR_INITIALIZATION"

Reply:
done